yasnippet.el

```diff
@@ -3286,58 +3300,29 @@ If so cleans up the whole snippet up."
 
 ;; Apropos markers-to-points:
 ;;
-;; This was found useful for performance reasons, so that an
-;; excessive number of live markers aren't kept around in the
-;; `buffer-undo-list'. However, in `markers-to-points', the
+;; This was found useful for performance reasons, so that an excessive
+;; number of live markers aren't kept around in the
+;; `buffer-undo-list'.  However, in `markers-to-points', the
 ;; set-to-nil markers can't simply be discarded and replaced with
-;; fresh ones in `points-to-markers'. The original marker that was
-;; just set to nil has to be reused.
+;; fresh ones in `points-to-markers'.  The original marker that was
+;; just set to nil has to be reused (why?).
```

Comment:
The experiment mentioned above disregards the "performance reasons" mentioned in the comment block, but I don't recall actually measuring any slowdown, so it makes sense to ignore it for simplicity's sake